backend/common/src/main/java/ai/verta/modeldb/common/authservice/AuthInterceptor.java

```diff
@@ -24,6 +26,21 @@
   public <R, S> Listener<R> interceptCall(
       ServerCall<R, S> call, Metadata requestHeaders, ServerCallHandler<R, S> next) {
     Context context = Context.current().withValue(METADATA_INFO, requestHeaders);
+
+    // To protect empty headers from request
```

Comment:
I don't get this comment.

Reply:
If user do not set the metadata headers in request then we are protecting request in the interceptor instead of passing to the backend api.